### PR TITLE
libexif: update to 0.6.22.

### PR DESCRIPTION
**Pull request by benalb**
This version is not in sourceforge but in the github repo. 
It fixes several CVEs, so we can get rid of the patches.
Also, claim maintainer.

### CODE REVIEW

File: srcpkgs/libexif/template [hunk 1]
```diff
@@ -1,16 +1,21 @@
...
 short_desc="EXIF file library"
-maintainer="Orphaned <orphan@voidlinux.org>"
+maintainer="Benjamín Albiñana <benalb@gmail.com>"
 license="LGPL-2.1-or-later"
-homepage="http://sourceforge.net/projects/libexif"
-distfiles="${SOURCEFORGE_SITE}/${pkgname}/${pkgname}-${version}.tar.bz2"
-checksum=16cdaeb62eb3e6dfab2435f7d7bccd2f37438d21c5218ec4e58efa9157d4d41a
-patch_args="-Np1"
+homepage="https://github.com/libexif/libexif"
+distfiles="https://github.com/libexif/libexif/archive/libexif-0_6_22-release.tar.gz"
```
Comment (Chocimier) [inline srcpkgs/libexif/template:13]: `version` should be actually used here, please change to `${version//./_}`

File: srcpkgs/libexif/template [hunk 2]
```diff
@@ -1,16 +1,21 @@
...
+checksum=46498934b7b931526fdee8fd8eb77a1dddedd529d5a6dbce88daf4384baecc54
+
+pre_configure() {
+	autoreconf -if
+	}
```
Comment (Chocimier) [inline srcpkgs/libexif/template:18]: indentation

File: srcpkgs/libexif/template (hunk 1 above)
Comment (benalb) [reply srcpkgs/libexif/template:13]: Done.

File: srcpkgs/libexif/template (hunk 2 above)
Comment (benalb) [reply srcpkgs/libexif/template]: fixed

File: srcpkgs/libexif/template (hunk 1 above)
Comment (Chocimier) [reply srcpkgs/libexif/template:13]: And remove old line please.

Comment (benalb) [reply srcpkgs/libexif/template]: fixed,

**Pull request merged**